daisy_workflows/image_import/ubuntu/translate.py

```diff
@@ -45,11 +45,17 @@
 ConnectPort 563
 '''
 
-partner_list = '''
+partner_repo = '''
 # Enabled for Google Cloud SDK
-deb http://archive.canonical.com/ubuntu {ubu_release} partner
+deb http://archive.canonical.com/ubuntu {ubuntu_release} partner
 '''
 
+# https://cloud.google.com/compute/docs/manage-os
+os_config_repo = '''
+# Enabled for GCE OS config agent
+deb http://packages.cloud.google.com/apt google-compute-engine-{ubuntu_release}-stable main
```

Comment:
same question here about repos and if we have discussed this change with canonical

Reply:
Similar to above, regarding import, I haven't talked with Canonical on this. Not sure about OS config overall.

Reply:
Ditto to comment regarding SUSE